package.json

```diff
@@ -12,6 +12,7 @@
     "aws-sdk": "^2.2.43",
     "boom": "^2.6.1",
     "csv-parser": "^1.5.0",
+    "decrypt-kms-env": "^2.0.1",
```

Comment:
@aaronlidman  I am using it to pass the envs to the app when it is runing in docker. 
`CMD eval $(./to-fix-backend/node_modules/.bin/decrypt-kms-env) && cd to-fix-backend/ && npm start`. I am  looking to move this   in the build repository.